src/lib.rs

```diff
@@ -1492,15 +1492,16 @@ fn write_general_subtrees(writer: DERWriter, tag: u64, general_subtrees: &[Gener
 impl Certificate {
 	/// Generates a new certificate from the given parameters.
 	///
-	/// If there is no key pair included, then a new key pair will be generated and used.
+	/// If there is no key pair included, then a new key pair will be randomly generated and used.
+	/// If you want to control the [`KeyPair`] or the randomness used to generate it, set it ahead of time before calling this function.
```

Comment:
I think this might be a touch more user friendly if we explicitly link to the params field in question:

```suggestion
	/// If you want to control the [`KeyPair`] or the randomness used to generate it, set the [`CertificateParams::key_pair`]
	/// field ahead of time before calling this function.
```